pontoon/batch/utils.py

```diff
@@ -30,6 +40,31 @@ def get_translations_info(translations, locale):
     return count, translated_resources, changed_entities
 
 
+def ftl_find_and_replace(string, find, replace):
+    """Replace text values in an FTL string.
+
+    :arg string string: a serialized FTL string
+    :arg string find: a string to search for, and replace, in translations
+    :arg string replace: what to replace the original string with
+
+    :returns: a serialized FTL string
+    """
+
+    def fun(value):
+        if type(value) == ast.TextElement:
+            value.value = value.value.replace(find, replace)
+        return value
+
+    # Parse FTL string and get AST
+    old_ast = parser.parse_entry(string)
+
+    # Recursively traverse the AST and perform find and replace on text values only
+    new_ast = old_ast.traverse(fun)
+
+    # Serialize updated AST
+    return serializer.serialize_entry(new_ast)
```

Comment:
This is just a nit, but the first and the last comment here kind of repeat exactly what's written in code, and the middle comment could be move to a docstring of the function passed to `traverse` :)